### PR TITLE
Forms

### PR DESCRIPTION
**Pull request by Strangeal**
In this PR, I added rails forms in view and the controller and other features according to the project requirements. Below are the features I added:

- Added a method called `current_user` in `ApplicationController` 
- Added logic inside `current_user` to return the first user from the `database`.
- Added forms to perform the following functions:
   - Create a new `post` on behalf of the `current_user`
   - Create a new `comment` on behalf of the `current_user`
- Allowed Users to add likes to Posts.

### CODE REVIEW

File: app/controllers/comments_controller.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+class CommentsController < ApplicationController
+  def new
+    @comment = Comment.new
+  end
+
+  def create
+    @comment = Comment.new(text: params[:text], author: current_user, post_id: params[:id])
+    p params
```
Comment (AnselemOdims) [inline app/controllers/comments_controller.rb]: - [x] Kindly see above suggestion

File: app/controllers/posts_controller.rb [hunk 2]
```diff
@@ -4,6 +4,20 @@ def index
     @posts = @user.posts
   end
 
+  def new
+    @post = Post.new
+  end
+
+  def create
+    @post = Post.new(author: current_user, title: params[:post][:title], text: params[:post][:text])
```
Comment (AnselemOdims) [inline app/controllers/posts_controller.rb]: - [x] [OPTIONAL] Good job implementing this, but please be aware that you can create a private method that can be reused. You can read more about it below

`https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/lessons/Controllers_lesson.md#:~:text=Pay%20special%20attention%20to%20the%20strong%20parameters%20description%20in%20Rails%20Guides`

**Review by AnselemOdims — APPROVED** [review]
### Hi  @Strangeal 🙋‍♀️, 
Good job implementing all the project requirements for this milestone/project.

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Use the tag @AnselemOdims for a quicker reply


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Strangeal** [comment]
Thank you

**Pull request merged**